src/utils/get-comment-registry.ts

```diff
@@ -422,9 +428,14 @@ export function attachCommentsToOutputNodes(
                     line: firstImport.loc?.end.line + commentHeight,
                 },
             };
+
             const moveDist =
                 originalLoc.start.line - newFirstImport.loc.start.line;
 
+            if (provideGapAfterTopOfFileComments) {
```

Comment:
I guess we also need to make sure that the top-of-file comment isn't a `prettier-ignore` comment.